infra/terraform/platform/main.tf

```diff
@@ -1,55 +1,44 @@
 terraform {
   backend "s3" {
     bucket = "terraform.analytics.justice.gov.uk"
-    key    = "dev/terraform.tfstate"
+    key    = "platform.tfstate"
```

Comment:
So, just to state the obvious/double check: The TF state would be in a sub "directory" in the S3 bucket called after the workspace name right?

Is that what you meant by saying "copying the state"? As in copying this `(dev|alpha)/terraform.tfstate` to `(dev/alpha)/platform.tfstate`? (Or something like that)

Reply:
(Yep)[https://www.terraform.io/docs/backends/types/s3.html#key] - according to that, the actual state object for `dev` will be at `/:env/dev/platform.tfstate`

"Copying the state" refers to copying the data in the existing state object (`dev/terraform.tfstate`) to the new one (`/:env/dev/platform.tfstate`) - although it turns out that `terraform state` has no `copy` command, only `mv`, so we'll need to either move the state data, or import existing resources into the new state object.